src/API/Site/Controllers/MerchantCenter/PriceBenchmarksController.php

```diff
@@ -94,17 +94,18 @@ protected function prepare_query_arguments( Request $request ): array {
 	 * @return callable
 	 */
 	protected function get_price_benchmarks_callback(): callable {
-		return function ( Request $request ) {
+		return function () {
 			try {
-				$response_data = $this->get_price_benchmarks_base_data(
-					$this->prepare_query_arguments( $request )
-				);
+				/** @var PriceBenchmarks $price_benchmarks */
+				$price_benchmarks = $this->container->get( PriceBenchmarks::class );
+
+				$response_data = $price_benchmarks->get_price_benchmarks_data();
```

Comment:
We'll still need to accept and prepare query args here and use those as the args for the get_price_benchmarks_data method.
```suggestion
				$response_data = $price_benchmarks->get_price_benchmarks_data( $this->prepare_query_arguments( $request ) );
```